onecgiar-pr-server/src/api/notification/notification.controller.ts

```diff
@@ -0,0 +1,86 @@
+import { Controller, Post, Body, Get, Param, Patch } from '@nestjs/common';
+import { NotificationService } from './notification.service';
+import { CreateAnnouncementNotificationDto } from './dto/create-notification.dto';
+import { UserToken } from '../../shared/decorators/user-token.decorator';
+import { TokenDto } from '../../shared/globalInterfaces/token.dto';
+import { ApiTags, ApiOperation, ApiResponse, ApiHeader } from '@nestjs/swagger';
+
+@ApiTags('Notifications')
+@ApiHeader({
+  name: 'auth',
+  description: 'the token we need for auth.',
+})
+@Controller()
+export class NotificationController {
+  constructor(private readonly notificationService: NotificationService) {}
+
+  @ApiOperation({ summary: 'Create a new announcement notification' })
+  @ApiResponse({
+    status: 201,
+    description: 'The notification has been successfully created.',
+  })
+  @ApiResponse({ status: 403, description: 'Forbidden.' })
```

Comment:
@JuanGuzman-io consider explaining why would you get a HTTP code 403 when using the endpoint